src/api/providers/fetchers/openrouter.ts

```diff
@@ -239,5 +239,10 @@ export const parseOpenRouterModel = ({
 		modelInfo.maxTokens = anthropicModels["claude-3-7-sonnet-20250219:thinking"].maxTokens
 	}
 
+	// Set horizon-alpha model to 32k max tokens
+	if (id === "openrouter/horizon-alpha") {
+		modelInfo.maxTokens = 32768
+	}
```

Comment:
Already told you, but I don't like this change, especially since it doesn't seem to work.